vars/test.yml

```diff
@@ -353,6 +360,16 @@ windows_product_key_upgrade: "XXXXX-XXXXX-XXXXX-XXXXX-XXXXX"
 #
 # gosc_dhcp_network: "VM Network"
 
+# 7. windows_update_install
+# For Windows testing only.
+# Windows shared folder for storing the .msu file. E.g. \\WINDOWS_SERVER_ADDRESS\SHARE_FOLDER
+windows_nfs_share: ''
+# The path of the .msu file in above Windows shared folder. E.g. WindowsClient\Windows11\23H2
+windows_msu_nfs_path: ''
```

Comment:
Can this variable name be changed to windows_nfs_msu_path? So it can be consistent with other variables.